flake.nix

```diff
@@ -66,8 +66,11 @@
           server = {
             description = "Initialize and launch the web server";
             exec = ''
-              nix run --no-warn-dirty .#update-json \
-              && nix run .#
+              export NIX_CONFIG='warn-dirty = false' \
```

Comment:
Why is this export defined?

Reply:
I did not bother to add `--no-warn-dirty` for all the Nix commands, so I just threw that in. The best solution for this would be to get the `NIX_CONFIG` environment variable defined for the devShell. I tried and failed.